structs.go

```diff
@@ -784,6 +784,20 @@ func (m *Member) Mention() string {
 	return "<@!" + m.User.ID + ">"
 }
 
+// A MemberEdit holds Member Field data for a member edit.
+type MemberEdit struct {
+	Nick  string   `json:"nick,omitempty"`
+	Roles []string `json:"roles,omitempty"`
+
+	// Cannot change below fields if user is not in voice channel
+	Mute bool `json:"mute,omitempty"`
+	Deaf bool `json:"deaf,omitempty"`
```

Comment:
There's no way to unmute people with `omitempty` present, as a `false` will be treated as an empty.

Reply:
Oy vey! You're right. :disappointed: Thanks.